experimental/dds/tree2/src/class-tree/tree.ts

```diff
@@ -16,7 +16,7 @@ import {
 } from "./schemaTypes";
 /**
  * Channel for a Tree DDS.
```

Comment:
Since this interface does more than `IChannel`, it could probably use more detailed docs